### PR TITLE
Translate 10. Open Science

### LINKED ISSUE #70: Translation: git-novice Episode 10
Translation: git-novice Section 10 "Open Science"
-----

* Please assign yourself (or add a comment) to the issue **before** you start translating, so other people know that you are translating.
* Once you have finished translating, send a pull request and reference this issue.

翻訳：git-novice セクション10 "Open Science"
-----

* あなたが翻訳し始めたと分かるように、翻訳を**始める前**にこのイシューを自分に割り振って(assign)、あるいはコメントを書いて下さい。
* 翻訳が終わったら、プルリクエストを開いて、このイシューを参照・リンクして下さい。

https://github.com/swcarpentry-ja/i18n/blob/b3225373bbf9b2f14b8882e4bc019cbd9dbad114/po/git-novice.ja.po#L7256

### PR DESCRIPTION
**Pull request by kozo2**

### CODE REVIEW

File: po/git-novice.ja.po [hunk 1]
```diff
@@ -7536,14 +7536,15 @@ msgid ""
 "but the reality is often more complicated.\n"
 "Normal practice today looks something like this:"
 msgstr ""
-"Free sharing of information might be the ideal in science,\n"
-"but the reality is often more complicated.\n"
+"科学では情報を自由に共有することが理想的かもしれませんが、\n"
+"現実はもっと複雑なことがよくあります。\n"
+"今日の通常の練習は次のようになります。"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: In this context, "practice" does not mean practice as in to improve a skill; it means "custom".
```suggestion
"最近のよくあるやり方は次のようになります。"
```

File: po/git-novice.ja.po [hunk 2]
```diff
@@ -7536,14 +7536,15 @@ msgid ""
...
 "Normal practice today looks something like this:"
 
 # unordered list
 #: git-novice/_episodes/10-open.md:23
 msgid "*   A scientist collects some data and stores it on a machine"
-msgstr "*   A scientist collects some data and stores it on a machine"
+msgstr "*   科学者はいくつかのデータを収集し、"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
msgstr "*   科学者は何らかのデータを収集し、"
```

File: po/git-novice.ja.po [hunk 3]
```diff
@@ -7569,46 +7570,40 @@ msgid ""
...
+"*   時間が経過します。\n"
+"*   ジャーナルは、科学者の分野の他の少数の人々によって匿名で書かれたレビュー\n"
+"    を科学者に送信します。\n"
+"    科学者はレビューした人を納得させるために自分の論文を改訂し、\n"
+"    その間に科学者は以前に書いたスクリプトを修正し、\n"
+"    論文を再提出するかもしれません。\n"
+"*   さらに時間が経過します。\n"
+"*   ついに論文が出版されます。\n"
+"    科学者のデータのオンラインコピーへのリンクが含まれている可能性があります、\n"
+"    ですが論文自体はペイウォールの背後にあるでしょう:\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"    が論文自体はペイウォールの背後にあるでしょう:\n"
```

File: po/git-novice.ja.po [hunk 4]
```diff
@@ -7638,31 +7633,25 @@ msgid ""
...
+"    [figshare](https://figshare.com/)や\n"
+"    [Zenodo](https://zenodo.org)\n"
+"    などのオープンアクセスリポジトリに保存され、独自の\n"
+"    [デジタルオブジェクト識別子](https://en.wikipedia.org/wiki/"
+"Digital_object_identifier) (DOI) が与えられます。\n"
+"    または、データはすでに公開されており、\n"
+"    [Dryad](https://datadryad.org/) に保存されています。\n"
+"*   科学者は、自分の研究を保持するためにGitHubに新しいリポジトリを作成します。\n"
+"*   科学者は分析を行うときに、スクリプト (場合によってはいくつかの出力ファイルも) への変更をそのリポジトリにプッシュします。\n"
+"    科学者はまた、自分の論文用のリポジトリを使用します;\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: She doesn't create another repository for the paper, but uses the same one for the paper (unfortunately, it's not clear what "use the repository for her paper" means)
```suggestion
"    科学者はまた、自分の論文用にそのリポジトリを使用します;\n"
```

File: po/git-novice.ja.po [hunk 5]
```diff
@@ -7676,15 +7665,15 @@ msgid ""
...
-"the more open work is,\n"
-"[the more widely it is cited and re-used](https://doi.org/10.1371/journal."
-"pone.0000308).\n"
-"However,\n"
-"people who want to work this way need to make some decisions\n"
-"about what exactly \"open\" means and how to do it. You can find more on the "
-"different aspects of Open Science in [this book](https://link.springer.com/"
-"book/10.1007/978-3-319-00026-8)."
+"このオープンモデルは発見を加速します:\n"
+"オープンな作品が多いほど、\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: "more" does not refer to number of research products (work), but how open they are.
```suggestion
"研究結果がオープンになっていればなっているほど、\n"
```

File: po/git-novice.ja.po [hunk 6]
```diff
@@ -7693,17 +7682,17 @@ msgid ""
 "it answers the \"how\" question\n"
 "by acting as a shareable electronic lab notebook for computational work:"
 msgstr ""
-"This is one of the (many) reasons we teach version control.\n"
-"When used diligently,\n"
-"it answers the \"how\" question\n"
-"by acting as a shareable electronic lab notebook for computational work:"
+"これは、バージョン管理を教える (多くの) 理由の1つです。\n"
+"コツコツと使うと、When used diligently,\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"コツコツと使うと、\n"
```

File: po/git-novice.ja.po [hunk 7]
```diff
@@ -7716,19 +7705,19 @@ msgid ""
...
-"    what and when. Every step is stamped with an identifier (the commit ID)\n"
-"    that is for most intents and purposes unique.\n"
-"*   You can tie documentation of rationale, ideas, and other\n"
-"    intellectual work directly to the changes that spring from them.\n"
-"*   You can refer to what you used in your research to obtain your\n"
-"    computational results in a way that is unique and recoverable.\n"
-"*   With a version control system such as Git, \n"
-"    the entire history of the repository is easy to archive for perpetuity."
+"    作業の概念的な段階が文書化されています。すべてのステップには、\n"
+"    ほとんどの意図と目的に固有の識別子 (コミットID) がスタンプされています。\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: A literal translation is not needed for this expression. コミットのSHAが必ずしもユーニクではないけど、極めて多くの場合はそうであることを言っている。
```suggestion
"    （ほとんどの場合）固有の識別子 (コミットID) がスタンプされています。\n"
```

File: po/git-novice.ja.po [hunk 8]
```diff
@@ -7758,15 +7746,15 @@ msgid ""
...
 msgstr ""
 ">\n"
-"> Ask one of your labmates to reproduce a result you recently obtained\n"
-"> using only what they can find in your papers or on the web.\n"
-"> Try to do the same for one of their results,\n"
-"> then try to do it for a result from a lab you work with."
+"> ラボメイトの1人に、あなたの論文やウェブで見つけることができるものだけを使用して、\n"
+"> あなたが最近得た結果を再現するように依頼してみましょう。\n"
+"> ラボメイトの結果の1つに対して同じことを行ってみてから、\n"
+"> あなたの研究由来の結果にそれを行ってみてください。"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: Need to specify "lab"
```suggestion
"> あなたの研究室由来の結果にそれを行ってみてください。"
```

**Review by joelnitta — COMMENTED** [review]
Thank you! Please see my comments and suggested edits.

**Comment by kozo2** [comment]
@joelnitta Thank you for your comments and suggested edits.
I checked and committed all of them.

**Pull request merged**